server/src/routes/Request.js

```diff
@@ -25,4 +30,15 @@ requestRouter.post("/create", checkUserId, verifyToken, createRequest);
 
 requestRouter.put("/priority", checkUserId, verifyToken, changePriority);
 
+requestRouter.post(
```

Comment:
Ok to use `POST` method here, but you may find this hard to maintain in the future. RESTFUL API convention usually has HTTP method describing the action and url path describing the resources. I.e `POST /request` is read as create a new request, `PATCH /request/:id` is update the request associated with this id, etc.